spec/models/world_location_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+describe WorldLocation do
+  describe :all do
+    it "returns a list of WorldLocations" do
+      allow(Rails).to receive_message_chain(:cache, :fetch).and_return([
+        { "format" => "World location", "details" => { "slug" => "United Kingdom" } },
+        { "format" => "World location", "details" => { "slug" => "Finland" } },
+      ])
```

Comment:
It's unusual to stub the Rails cache since it works transparently in testing. You should instead mock the API.

At the moment all we're testing is that Rails.cache returns data and that's not really telling us anything here.